### PR TITLE
Node6hotfixes/dot net core cliv2

### PR DESCRIPTION
**Pull request by ddivanshu**
**Task name**: dotnetcorecliv2

**Description**: <Describe your changes here>

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DotNetCoreCLIV2/Tests/L0.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ describe('DotNetCoreExe Suite', function () {
     });
 
     it('fails if the dotnet tool is not found', (done: MochaDone) => {
+        this.timeout(5000);
```
Comment (bishal-pdMSFT) [inline Tasks/DotNetCoreCLIV2/Tests/L0.ts:16]: is this required?

File: Tasks/DotNetCoreCLIV2/restorecommand.ts [hunk 2]
```diff
@@ -119,7 +117,7 @@ export async function run(): Promise<void> {
         }
 
         // Setting creds in the temp NuGet.config if needed
-        nuGetConfigHelper.setAuthForSourcesInTempNuGetConfig();
+        nuGetConfigHelper.setAuthForSourcesInTempNuGetConfig();
```
Comment (bishal-pdMSFT) [inline Tasks/DotNetCoreCLIV2/restorecommand.ts:120]: remove extra space

**Review by bishal-pdMSFT — APPROVED** [review]

File: Tasks/DotNetCoreCLIV2/Tests/L0.ts (hunk 1 above)
Comment (ddivanshu) [reply Tasks/DotNetCoreCLIV2/Tests/L0.ts:16]: yes this particular test was failing.In existing test cases also we have mentioned timeout of 1000 took reference from it

File: Tasks/DotNetCoreCLIV2/restorecommand.ts (hunk 2 above)
Comment (ddivanshu) [reply Tasks/DotNetCoreCLIV2/restorecommand.ts]: done

**Pull request merged**